### DISCUSSION TITLE
How to download certificate data?

### DISCUSSION THREAD

**Discussion by Lancelee92** [shibayan/appservice-acmebot#357] [Q&A] [answered]
The api returned url (https://[app-name].azurewebsites.net/api/certificate/xxxxxxxxxxxxx) in location header.
But when I try with get request to the returned link (https://[app-name].azurewebsites.net/api/certificate/xxxxxxxxxxxxx),
it response with empty body and status 200.

My requirement is that i need to get the data for the ssl certificate, is it possible to do this?

Sorry if it seems like a stupid question, just starting to play with ssl certification.

**Comment by shibayan** [comment, marked as answer]
Certificates added to the App Service cannot be exported. If you need to export the certificate, please use Key Vault Acmebot.
https://github.com/shibayan/keyvault-acmebot
